Lo-Agency/polaris
# user dashboard UI

# Description
This pr make landing page that guid you to login page and create dashboard page when login to the webiste.
create dashboard for every user that can show every workspace they are invited to and user can create every entitiy she wants.

## Related issues
Fixes [#138](https://quire.io/w/Caspian_Tamin/138/user_dashboard_ui?view=tree)

## Type of change


# How Has This Been Tested?
1. go to home page
2. then click on login button and go to login page
3. login to the website
4. click on dashboard button then go to your panel
5. if you are invited to anybodys workspace you can see their roadmap or you can click on (My workspace) to see the roadmaps you created before.

# Checklist:

- x ] My code follows the style guidelines of this project
